Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -609,6 +609,379 @@ Start Safari on the iOS device. Run the above Python script on your connected ho
 
 Of course, this example illustrates only one of the things you can do with Frida. To unlock the tool's full potential, you should learn to use its [JavaScript API](https://www.frida.re/docs/javascript-api/ "Frida JavaScript API reference"). The documentation section of the Frida website has a [tutorial](https://www.frida.re/docs/ios/ "Frida Tutorial") and [examples](https://www.frida.re/docs/examples/ios/ "Frida examples") for using Frida on iOS.
 
+##### Process Exploration
+
+When testing an app, process exploration can provide the tester with deep insights into the app process memory. It can be achieved via runtime instrumentation and allows to perform tasks such as:
+
+- Retrieving the memory map and loaded libraries.
+- Searching for occurrences of certain data.
+- After doing a search, obtaining the location of a certain offset in the memory map.
+- Performing a memory dump and inspect or reverse engineer the binary data _offline_.
+- Reverse engineering a binary or Framework while it's running.
```

Comment:
this seems to be a bit more repetitive.. maybe some of the general testing can be centralized later? (not now...)

Reply:
Totally agree :)